readme.md

```diff
@@ -13,6 +13,17 @@ pip install ovos-tts-plugin-server
 ```json
   "tts": {
     "module": "ovos-tts-plugin-server",
-    "ovos-tts-plugin-server": {"host": "https://0.0.0.0:9666"}
+    "ovos-tts-plugin-server": {"host": "https://0.0.0.0:9666"},
+    "host": "https://tts.smartgic.io/piper",
+    "verify_ssl": "true"
  }
 ```
+
+If using a Piper TTS server since version 0.0.1a9, you can use the
+`/v2` endpoint to take advantage of newer features. Be sure to include that
+in the `host` value if you'd like to use it. Example: `https://tts.smartgic.io/piper/v2`
```

Comment:
not sure thats quite right, the "v2" should not be manually included in url

the request is built from the base url and adds either /v2/synthesize or /synthesize to it, but the config takes the base_url

the example as is would break the /status endpoint and become `https://tts.smartgic.io/piper/v2/status` and  `https://tts.smartgic.io/piper/v2/v2/synthesize`

Reply:
Corrected the readme